src/ucp/wireup/address.c

```diff
@@ -505,7 +505,10 @@ ucp_address_unpack_iface_attr(ucp_worker_t *worker,
     packed                = ptr;
     iface_attr->priority  = packed->prio_cap_flags & UCS_MASK(8);
     iface_attr->overhead  = packed->overhead;
-    iface_attr->bandwidth = packed->bandwidth;
+    /* Get only the absolute value of the bandwidth considering the received
+     * bandwidth as a dedicated one to be compatible with the old versions
+     * which could send "dedicated - shared" bandwidth value */
+    iface_attr->bandwidth = fabs(packed->bandwidth);
```

Comment:
many should divide "shared" by ppn here?

Reply:
fixed